scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -2213,15 +2326,28 @@ then
             fi
         fi
 
-        if [[ $keyring_plugin -eq 0 && -n $transition_key ]]; then
+        if [[ -n $transition_key && $keyring_plugin -eq 0 && $keyring_component_enabled -eq 0 ]]; then
             wsrep_log_error "******************* FATAL ERROR ********************** "
-            wsrep_log_error "FATAL: DONOR is configured to use keyring_plugin" \
+            wsrep_log_error "FATAL: DONOR is configured to use keyring component/plugin" \
                             "(file/vault) but JOINER is not"
             wsrep_log_error "Line $LINENO"
             wsrep_log_error "****************************************************** "
             exit 32
         fi
 
+        if [ $keyring_component_enabled -eq 1 ]; then
+            # Handle joiner side component specific cnf file here
+            if [ ! -e $keyring_config_file ]; then
```

Comment:
Is it possible? we set above keyring_component_enabled=1 if there is mainfiest and config

Reply:
That's true. I have removed this check.

Reply:
Done